packages/central-server/src/apiV2/meditrakApp/getLeaderboardList.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import { respond } from '@tupaia/utils';
+import { getLeaderboard } from '../../social';
+import { allowNoPermissions } from '../../permissions';
+
+export const getLeaderboardList = async (req, res) => {
```

Comment:
I created this because the `meditrak-app-server` had the same logic as `datatrak-web-server` was going to, and much of this was already in `central-server`. Are you okay with that @rohan-bes ? I didn't like the idea of duplicating the exact same logic

Reply:
I won't speak for Rohan but From my point of view, this is a good call 👍